kong/plugins/galileo/buffer.lua

```diff
@@ -111,7 +112,7 @@ _send = function(premature, self, to_send)
   local client = http.new()
   client:set_timeout(self.connection_timeout)
 
-  local ok, err = client:connect(self.host, self.port)
+  local ok, err = connect(client, self.host, self.port)
```

Comment:
How do you see that? the `connect` method is metatable property of the socket userdata. I don't think we should mess with that. Or did you have something different in mind?